src/test/java/com/google/api/generator/engine/format/JavaDocCommentTest.java

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.format;
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+import com.google.api.generator.engine.format.JavaDocComment.ParamPair;
+
+import org.junit.Test;
+
+public class JavaDocCommentTest {
```

Comment:
Could we please add more test cases from the [baseline](https://github.com/googleapis/gapic-generator/blob/master/src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline#L2305) file? Specifically, tags like `<li>` (and others), special characters, etc.

Reply:
yes sure, tags including <ul> <ol> <li> will be covered in javaDoccomment and also unit tests.